x11-packages/arc-gtk-theme/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jnsh/arc-theme/
+TERMUX_PKG_DESCRIPTION="A flat theme with transparent elements"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=20221218
+TERMUX_PKG_SRCURL=https://github.com/jnsh/arc-theme/releases/download/${TERMUX_PKG_VERSION}/arc-theme-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=264570cc90a13b88d181334fb325f99a6ddc2a45392aa9ed7dba312aea042bb0
+TERMUX_PKG_DEPENDS="sassc"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_AUTO_UPDATE=false
+
+termux_step_configure(){
+    termux_setup_meson
```

Comment:
This fails the package linter check.
Please use hard tabs for indentation for shell scripts,
as per the coding guidelines.
https://github.com/termux/termux-packages/wiki/Coding-guideline#script-formatting

Reply:
already pushed fix. thanks!